swarmer/src/main/kotlin/com/gojuno/swarmer/Emulators.kt

```diff
@@ -363,3 +373,41 @@ private fun createdEmulators(args: Commands.Start, timeout: Pair<Int, TimeUnit>
                             .map { it.trim() }
                             .toSet()
                 }
+
+private fun connectedAdbDevicesNoModel(): Observable<Set<AdbDevice>> = process(listOf(adb, "devices"), unbufferedOutput = true)
```

Comment:
This is largely [copied from commander](https://github.com/gojuno/commander/blob/a15e3a49725a790a094f4bb996f70d69fb192743/android/src/main/kotlin/com/gojuno/commander/android/Adb.kt#L38) minus the call to `deviceModel()` because 1) we don't actually care about the device model, and 2) it will cause `connectedAdbDevicesNoModel()` to fail if one of the devices is disconnected during, which is highly likely while we're killing them.